1-Authentication/2-sign-in-b2c/README.md

```diff
@@ -54,70 +71,78 @@ From your shell or command line:
+
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure Active Directory B2C** service.
+1. Select the **App Registrations** blade on the left, then select **New registration**.
+1. In the **Register an application page** that appears, enter your application's registration information:
+    1. In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `ms-identity-javascript-c1s2`.
+    1. Under **Supported account types**, select **Accounts in any identity provider or organizational directory (for authenticating users with user flows)**
+    1. Select **Register** to create the application.
+1. In the **Overview** blade, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select the **Authentication** blade to the left.
+1. If you don't have a platform added, select **Add a platform** and select the **Single-page application** option.
+    1. In the **Redirect URI** section enter the following redirect URIs:
+        1. `http://localhost:6420`
+        1. `http://localhost:6420/redirect`
+    1. Click **Save** to save your changes.
+
+##### Configure the client app (ms-identity-javascript-c1s2) to use your app registration
 
 Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
 
 > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
 
 1. Open the `App\authConfig.js` file.
-1. Find the key `clientId` and replace the existing value with the application ID (clientId) of the `ms-identity-javascript-tutorial-c1s2` application copied from the Azure portal.
-1. Find the key `redirectUri` and replace the existing value with the base address of the `ms-identity-javascript-tutorial-c1s2` app (by default `http://localhost:6420`).
-1. Find the key `postLogoutRedirectUri` and replace the existing value with the base address of the `ms-identity-javascript-tutorial-c1s2` app and the **signout** page that the app will redirect to e.g. `http://localhost:6420/signout`.
+1. Find the key `clientId` and replace the existing value with the application ID (clientId) of `ms-identity-javascript-c1s2` app copied from the Azure portal.
+
+To setup your B2C user-flows, do the following:
 
-1. Open the `App\policies.js` file.
 1. Find the key `names` and populate it with your policy names e.g. `signUpSignIn`.
 1. Find the key `authorities` and populate it with your policy authority strings e.g. `https://<your-tenant-name>.b2clogin.com/<your-tenant-name>.onmicrosoft.com/b2c_1_susi`.
 1. Find the key `authorityDomain` and populate it with the domain portion of your authority string e.g. `<your-tenant-name>.b2clogin.com`.
 
-## Running the sample
+
+### Step 4: Running the sample
 
 ```console
-    cd 1-Authentication/2-sign-in-b2c
+    cd 1-Authentication\2-sign-in-b2c
     npm start
```

Comment:
>npm start [](http://example.com/codeflow?start=4&length=9)

it never starts unless i do the
npm install
first :unamused:

Reply:
The [second step ](cd 1-Authentication\2-sign-in-b2c)in the setup is to run npm insall